apps/web/components/Embed.tsx

```diff
@@ -1258,20 +1636,23 @@ type EmbedButtonProps<T> = {
   children?: React.ReactNode;
   className?: string;
   as?: T;
+  eventId?: number;
 };
 
 export const EmbedButton = <T extends React.ElementType>({
   embedUrl,
   children,
   className = "",
   as,
+  eventId,
   ...props
 }: EmbedButtonProps<T> & React.ComponentPropsWithoutRef<T>) => {
   const router = useRouter();
   className = classNames("hidden lg:inline-flex", className);
   const openEmbedModal = () => {
     goto(router, {
       dialog: "embed",
+      eventId: eventId ? eventId.toString() : "",
```

Comment:
Have added eventId as a needed parameter as based on that we're querying the selectedDatesAndTimes for this eventType